packages/expo-file-system/src/next/FileSystem.types.ts

```diff
@@ -146,3 +146,71 @@ export declare class File {
    */
   md5: string | null;
 }
+
+export declare class Paths {
+  /*
+   * A property containing the document directory – a place to store files that are safe from being deleted by the system.
```

Comment:
```suggestion
   * A property containing the document directory – a place to store permanent files that won't be removed by the system when the device is low on storage.
```
I didn't like `files that are safe` part.